packages/hash/shared/src/blockPaths.json

```diff
@@ -1,11 +1,11 @@
 {
-  "https://block.blockprotocol.org/paragraph": "https://blockprotocol-git-main-hashintel.vercel.app/blocks/@hash/paragraph",
-  "https://block.blockprotocol.org/header": "https://blockprotocol-git-main-hashintel.vercel.app/blocks/@hash/header",
-  "https://block.blockprotocol.org/person": "https://blockprotocol-git-main-hashintel.vercel.app/blocks/@hash/person",
-  "https://block.blockprotocol.org/image": "https://blockprotocol-git-main-hashintel.vercel.app/blocks/@hash/image",
-  "https://block.blockprotocol.org/table": "https://blockprotocol-git-main-hashintel.vercel.app/blocks/@hash/table",
-  "https://block.blockprotocol.org/divider": "https://blockprotocol-git-main-hashintel.vercel.app/blocks/@hash/divider",
-  "https://block.blockprotocol.org/embed": "https://blockprotocol-git-main-hashintel.vercel.app/blocks/@hash/embed",
-  "https://block.blockprotocol.org/code": "https://blockprotocol-git-main-hashintel.vercel.app/blocks/@hash/code",
-  "https://block.blockprotocol.org/video": "https://blockprotocol-git-main-hashintel.vercel.app/blocks/@hash/video"
+  "https://block.blockprotocol.org/paragraph": "https://blockprotocol.org/blocks/@hash/paragraph",
+  "https://block.blockprotocol.org/header": "https://blockprotocol.org/blocks/@hash/header",
+  "https://block.blockprotocol.org/person": "https://blockprotocol.org/blocks/@hash/person",
+  "https://block.blockprotocol.org/image": "https://blockprotocol.org/blocks/@hash/image",
+  "https://block.blockprotocol.org/table": "https://blockprotocol.org/blocks/@hash/table",
+  "https://block.blockprotocol.org/divider": "https://blockprotocol.org/blocks/@hash/divider",
+  "https://block.blockprotocol.org/embed": "https://blockprotocol.org/blocks/@hash/embed",
+  "https://block.blockprotocol.org/code": "https://blockprotocol.org/blocks/@hash/code",
+  "https://block.blockprotocol.org/video": "https://blockprotocol.org/blocks/@hash/video"
```

Comment:
Can we replace the left side here too, and do a search & replace for any instances of the old path? This'll get rid of all the fake URLs.

e.g. this becomes 
```json5
{ 
  "https://blockprotocol.org/blocks/@hash/paragraph": "https://blockprotocol.org/blocks/@hash/paragraph"
  ... 
}
```

There are instances of the old URLs in `mockData.ts` and `createPage.ts` we should replace - otherwise we're creating types for e.g. a paragraph entity with different `componentId`s

With that done we might be almost at the point where we can just remove this entirely - would welcome @nathggns's views

Appreciate this is slightly expanding the scope of the task, but it'd be helpful to get rid of these confusing fake URLs now that we're using the `componentId` more and more.